docs/architecture/resilience.txt

```diff
@@ -437,6 +493,59 @@ the node with the original primary shard rejoins the cluster, CrateDB has no
 way of distinguishing correct and incorrect data, so that data replaced with
 data from the new primary shard.
 
+Make table creation resilient to closing and full cluster crashes
```

Comment:
Not sure if it makes sense to document issues that have been resolved quite a while ago?

Reply:
I wanted to make this complete. Let's drop all resolved issues from the document then.

Reply:
I wouldn't drop existing old one as people may know them and want to see the update.

Reply:
I specifically made sure to compare all issues with the ES document. I don't feel good with merging this in an incomplete state. I'm ok with removing the "fixed issues" section but not presenting it incomplete.

Reply:
I don't have any strong opinion on that. If this issue was encountered in our older CrateDB versions, I don't mind to mention it here as fixed.